### ISSUE TITLE
Manual infusion stabilization [Suggestion]

### ISSUE THREAD

**Issue by FanghanHu** [Azanor/thaumcraft-beta#1120]
"During my first few attempts I noticed that infusion is a chaotic process that is dangerous and unpredictable, it frequently cause instability to the surrounding aura, tipping the balance of the basic elements, and causing all kinds of mayhem. Some detectable effects are: sudden burst of physical force,r random potentia leak in the form of lightning and most noticeable of them all, the creation of massive amount of flux. This got to be stopped before my lab is ruined."

After Research:
"Because of the infusion process is so unpredictable, my preparations provided very little help other than slowing down the inevitable. There is not very much I can do before the infusion starts, so I will have to deal with the instability as it happens. As a brilliant thaumaturge as I am, I have largely simplified the  instability into a treatable model, and I have created a small device to detect and neutralize it. All I have to do is to place a few of this device around the altar and wait, when it detects any unusual behavior of the aura, it will light up and change color depends on the aspect and amount of the essentia that is causing problem, then I will place a few crystal of opposite aspect on it. the device should then start breaking down the crystal and neutralize the problem causing essentia, thus stabilize the infusion matrix"

tl;dr
During the infusion, a few invisible, shifting points of instability will spawn, amount and magnitude should differ depends on how dangerous the infusion recipe is(0 or 1 for negligible, 2 for moderate, 3 for dangerous). I will reference these points of instability as POI from now on. 

Each POI should have the following attribute: direction, aspect, magnitude, magnitude increment, increment time left.

Each POI will cause the altar become more unstable/stable depends on how high their magnitude is, at low magnitude, the POI will make alter more stable, and at high magnitude it will make altar more unstable, this should replace the passive stability loss of the recipe.
When the infusion starts, each POI will start at 0 magnitude, and slowly build up magnitude depends on "magnitude increment", and it will stop when "increment time left" runs out.

Each POI will start at a random direction and it will SLOWLY change its direction as the infusion goes on.
Devices that is placed close to a POI's direction will react to the POI. depends on how close the device is, it will shine brighter and effect the POI better.
Each device's RGB value should be effected by multiple POIs, because each device is not only effected and effecting a single POI.
When spawning POIs, they should be spawned relatively far away from each other, and their movement speed should be the same so they keep a distant from each other, this way players won't be fixing one problem and causing another at the same time all the time.

1 crystal will be consumed every few seconds, and a flat amount of magnitude will be removed or added to the POI depends on if the player provided the correct type of crystal, the amount of magnitude should be effected by how close the device is to the POI's direction, a POI can be out of reach of a device so anything done to the device should have no impact to the POI at all. For an aer aspect POI, player should provide the terra crystal, and aqua to Ignis, ordo to perditio, POIs should only spawn with these basic aspects.

Player needs to follow the clue devices is providing them and provide different crystals to stabilize the process. Infusion should take way longer than it is right now, and each time player do the right thing, it will feel more rewarding.

**Comment by Azanor** [comment]
This issue was moved to Azanor/thaumcraft-suggestions#341
